README.md

```diff
@@ -37,9 +37,39 @@ y = istft_extractor.forward(real, imag, x.shape[-1])	# (batch_size, samples_num)
 ```
 
 # More examples
-```
+
+```python
```

Comment:
Why python is needed here?

Reply:
Enables python syntax highlighting for some chosen language. Otherwise, it is a bit hard to see.
E.g.,:
Before

```
for i in range(100):
    print(i)
```

after

```python
for i in range(100):
    print(i)
```

The [docs](https://docs.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks).

Can also be avoided.

Reply:
Got it! Thanks!